crates/release_plz/src/args/mod.rs

```diff
@@ -32,6 +35,8 @@ pub enum Command {
     /// - publish the package to the cargo registry.
     /// You can run this command in the CI on every commit in the main branch.
     Release(Release),
+    /// Generate command autocompletions for various shells
```

Comment:
```suggestion
    /// Generate command autocompletions for various shells.
```
Sorry for the nitpicking

Reply:
I feel ya :)